src/features/advanced/inv-compress-inventory-info.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  applyScopedClassCssRule,
+  applyScopedCssRule,
+} from '@src/infrastructure/prun-ui/refined-prun-css';
+import classes from './inv-compress-inventory-info.module.css';
+import css from '@src/utils/css-utils.module.css';
+import ContextControls from '@src/components/ContextControls.vue';
+import { getInvStore } from '@src/core/store-id';
+import { sitesStore } from '@src/infrastructure/prun-api/data/sites';
+import {
+  getEntityNaturalIdFromAddress,
+  getLocationLineFromAddress,
+} from '@src/infrastructure/prun-api/data/addresses';
+import { warehousesStore } from '@src/infrastructure/prun-api/data/warehouses';
+import { shipsStore } from '@src/infrastructure/prun-api/data/ships';
+
+async function onTileReady(tile: PrunTile) {
+  if (!tile.parameter) {
```

Comment:
This `if` was initially added here to prevent running in plain `INV` command without parameters (my guess). But now it's safe to remove this `if` because we await for `C.StoreView.container`. The inventory lists (like in plain `INV`) wont get past this await.